anxcloud/provider.go

```diff
@@ -32,14 +32,16 @@ func Provider() *schema.Provider {
 			},
 		},
 		ResourcesMap: map[string]*schema.Resource{
-			"anxcloud_virtual_server":     resourceVirtualServer(),
-			"anxcloud_vlan":               resourceVLAN(),
-			"anxcloud_network_prefix":     resourceNetworkPrefix(),
-			"anxcloud_ip_address":         resourceIPAddress(),
-			"anxcloud_tag":                resourceTag(),
-			"anxcloud_dns_zone":           resourceDNSZone(),
-			"anxcloud_dns_record":         resourceDNSRecord(),
-			"anxcloud_lbaas_loadbalancer": resourceLBaaSLoadBalancer(),
+			"anxcloud_virtual_server":        resourceVirtualServer(),
+			"anxcloud_vlan":                  resourceVLAN(),
+			"anxcloud_network_prefix":        resourceNetworkPrefix(),
+			"anxcloud_ip_address":            resourceIPAddress(),
+			"anxcloud_tag":                   resourceTag(),
+			"anxcloud_dns_zone":              resourceDNSZone(),
+			"anxcloud_dns_record":            resourceDNSRecord(),
+			"anxcloud_lbaas_loadbalancer":    resourceLBaaSLoadBalancer(),
+			"anxcloud_kubernetes_cluster":    resourceKubernetesCluster(),
+			"anxcloud_kubernetes_kubeconfig": resourceKubernetesKubeconfig(),
```

Comment:
@toothstone what was our reason to have the kubeconfig only retrieved on demand instead of always storing it on the cluster?

Reply:
It is extremely sensitive information, and we tried to minimize its exposure where possible.

Reply:
Not that I know, no.

Reply:
maybe that's something to reconsider then, but not blocking this PR.

Reply:
Yeah as far as I remember, there was a good amount of "let's try out how these GA actions work" in the mix as well.